### PR TITLE
WEB-705 Clicking on Thumbnail image opens a Gallery

### PR DESCRIPTION
**Pull request by red-runner**
[Related Jira issue](https://lodgify.atlassian.net/browse/WEB-705)

### What **one** thing does this PR do?
* Enables `Pictures` to open the `Gallery` component when a `Thumbnail` is clicked.
* Removes `LightBox` as a component to avoid a duplicate copy of `Gallery`.

### Any other notes
gallery
ss

### CODE REVIEW

File: src/components/property-page-widgets/Pictures/component.js [hunk 1]
```diff
@@ -23,19 +28,41 @@ export const Component = ({
...
+      label: descriptionText,
+    }));
+
+  return (
+    <Grid>
+      <GridColumn width={12}>
+        <Heading>{headingText}</Heading>
+      </GridColumn>
+      <GridRow>
+        {getFirstNItems(
```
Comment (huvber) [inline src/components/property-page-widgets/Pictures/component.js]: I hope this function doesn't just do slice 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice

Comment (red-runner) [reply src/components/property-page-widgets/Pictures/component.js]: Although, we can refactor this in another PR. This is not in scope.
```
export const getFirstNItems = (number, items) => items.slice(0, number);
```

Comment (huvber) [reply src/components/property-page-widgets/Pictures/component.js]: As you wish. I think we will forget it

File: src/components/media/Gallery/component.js [hunk 2]
```diff
@@ -1,26 +1,67 @@
...
+import { Slideshow } from 'media/Slideshow';
 import { testidFactory } from 'utils/testid';
 
 const TEST_ID_PREFIX = 'gallery';
 
 const testid = testidFactory(TEST_ID_PREFIX);
 
 /**
- * A gallery displays a collection of images in a modal.
+ * A Gallery which displays an image or a series of images in a modal.
```
Comment (Jonnygwi) [inline src/components/media/Gallery/component.js]:
```suggestion
 * A gallery displays an image or a series of images in a light box.
```

File: src/styles/semantic/themes/default/lodgify-ui-components/gallery.variables [hunk 3]
```diff
@@ -0,0 +1,26 @@
+/*--------------------------
+  LightBox
```
Comment (Jonnygwi) [inline src/styles/semantic/themes/default/lodgify-ui-components/gallery.variables]:
```suggestion
  Gallery
```

File: src/components/media/Gallery/component.js [hunk 4]
```diff
@@ -34,6 +75,8 @@ Component.propTypes = {
       url: PropTypes.string.isRequired,
     })
   ).isRequired,
-  /** The element to be clicked to display the gallery. */
-  trigger: PropTypes.node.isRequired,
+  /** The index used to define the image shown when the Slideshow is opened. */
```
Comment (Jonnygwi) [inline src/components/media/Gallery/component.js]:
```suggestion
  /** The index used to define the image shown when the slideshow is opened. */
```

File: src/components/media/Gallery/component.js [hunk 5]
```diff
@@ -1,26 +1,67 @@
...
+};
+
 Component.propTypes = {
-  /** The images to display.  */
-  images: PropTypes.arrayOf(
+  /** Is the Gallery in an open state. */
+  isOpen: PropTypes.bool,
+  /** The function called when the closed event happens. */
+  onClose: PropTypes.func,
+  /**  The images to display inside the Slideshow. */
```
Comment (Jonnygwi) [inline src/components/media/Gallery/component.js]:
```suggestion
  /**  The images to display inside the slideshow. */
```

File: src/components/media/Gallery/component.js [hunk 6]
```diff
@@ -1,26 +1,67 @@
...
+  trigger: null,
+  isOpen: undefined,
+  onClose: undefined,
+  startIndex: 0,
+};
+
 Component.propTypes = {
-  /** The images to display.  */
-  images: PropTypes.arrayOf(
+  /** Is the Gallery in an open state. */
```
Comment (Jonnygwi) [inline src/components/media/Gallery/component.js]:
```suggestion
  /** Is the gallery in an open state. */
```

File: src/components/media/Slideshow/component.js [hunk 7]
```diff
@@ -123,4 +133,6 @@ Component.propTypes = {
   isShowingThumbnails: PropTypes.bool,
   /** Is the slideshow fitting the height of it's container. */
   isStretched: PropTypes.bool,
+  /** The index used to define the image shown when the Slideshow is opened. */
```
Comment (Jonnygwi) [inline src/components/media/Slideshow/component.js]:
```suggestion
  /** The index used to define the image shown when the slideshow is opened. */
```

**Pull request merged**